app/public/javascripts/passValid.js

```diff
@@ -0,0 +1,13 @@
+function ValidatePassword (){
+    const myForm = document.getElementById('myForm');
+  const password = myForm.elements['password'].value;
+  const confirm_password = myForm.elements['confirm_password'].value;
+
+  if(password !== confirm_password){
+    alert("Please make sure passwords match.");
+        return false;
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8443809)
```suggestion
    return false;
```